src/v/kafka/server/group.cc

```diff
@@ -102,6 +102,8 @@ group::group(
   , _ctx_txlog(cluster::txlog, *this)
   , _md_serializer(std::move(serializer))
   , _enable_group_metrics(group_metrics)
+  , _abort_interval_ms(config::shard_local_cfg()
```

Comment:
This thing where we have two group constructors which vary only 1 parameter is probably an anti-pattern. Instead it could be a `std::variant` and then the constructors could share most initialization, with an `if` in the body for the 4-5 params which depend on which value is in the variant.

I didn't want to do that here though.